style/components/text.css

```diff
@@ -0,0 +1,21 @@
+h1 {
+  text-align: center;
+  background-color: #262626;
+  color: white;
+  padding: 15px;
+  font-size: 40px;
+  margin-bottom: 20px;
+}
+h3 {
+  text-align: right;
+  margin-top: -140px;
+  font-size: 20px;
+  margin-right: 7px;
+}
+h4 {
+  text-align: right;
+  margin-right: 7px;
+  margin-top: -100px;
+  font-style: italic;
+  font-size: 14px;
+}
```

Comment:
Sounds good to use a separate file for this! I think you could also consider to move it into the base.css, but this is entirely up to you!